### ISSUE TITLE
Split-screen shell

### ISSUE THREAD

**Issue by jessrosenfield** [warpdotdev/warp#14]
**Is your feature request related to a problem? Please describe.**
Sometimes you want to view shell sessions simultaneously. Two ways I typically go about this are via iTerm's split-pane feature or via tmux. Unfortunately, I lose some of the benefits baked into Warp when using tmux.

**Describe the solution you'd like**
Any sort of solution that addresses this use case! A several come to mind: some sort of tmux integration, tmux alternative, or native support for split screen functionality

**Comment by alokedesai** [comment]
Thanks for filing! This is on our roadmap--as of right now we plan on implementing this natively similar to iTerm

**Comment by chrissmartin** [comment]
I would love this feature to be implemented on priority. As running/monitoring bunch of servers is a pain without split panes. Also would like to request the mouse hover on pane to activate it kinda feature on iTerm, with shadow/bg color on inactive panes.

**Comment by ajtrichards** [comment]
Upvoted - would really love to see this feature become part of Warp. I use this feature extensively in my workflows and it would be the icing on the cake for Warp 👍

**Comment by alokedesai** [comment]
Closing this out, since this just launched with today's release! 

![Image 2021-09-29 at 2 06 04 PM](https://user-images.githubusercontent.com/4110292/135324501-d2a6aae6-787c-4491-ba66-c334f2b86844.jpg)

**Comment by ned-si** [comment]
> https://docs.warp.dev/help/known-issues
> 
> Still references this as a known issue. Might be worth updating the site!

Indeed. But another issue could be open for tmux support as, so far, running tmux inside warp removes all the "block" concept (and the advantages going with it).

**Comment by elviskahoro** [comment]
@ned-si I think these two issues are keeping track of most of the tmux related requests:
https://github.com/warpdotdev/warp/issues/42
https://github.com/warpdotdev/warp/issues/168
